stories/produksjonssystem/AccordionExample2.jsx

```diff
@@ -8,7 +8,7 @@
 
 import React, { Component, Fragment } from 'react';
 import { Accordions, AccordionSection } from '@ndla/accordion';
-import Button from '@ndla/button';
+import {ButtonV2 } from '@ndla/button';
```

Comment:
Antar det er dette prettier klager på.
```suggestion
import { ButtonV2 } from '@ndla/button';
```